types/v56/types.go

```diff
@@ -2812,3 +2821,39 @@ type VmAffinityRules struct {
 	Link           *Link             `xml:"Link,omitempty"` //
 	VmAffinityRule []*VmAffinityRule `xml:"VmAffinityRule,omitempty"`
 }
+
+// ControlAccessParams specifies access controls for a resource.
+type ControlAccessParams struct {
+	XMLName             xml.Name           `xml:"ControlAccessParams"`
+	Xmlns               string             `xml:"xmlns,attr"`
+	IsSharedToEveryone  bool               `xml:"IsSharedToEveryone"`            // If true, the resource is shared with everyone in the organization. Defaults to false.
+	EveryoneAccessLevel *string            `xml:"EveryoneAccessLevel,omitempty"` // If IsSharedToEveryone is true, this element must be present to specify the access level. for all members of the organization. One of: FullControl Change ReadOnly
+	AccessSettings      *AccessSettingList `xml:"AccessSettings,omitempty"`      // The access settings to be applied if IsSharedToEveryone is false. Required on create and modify if IsSharedToEveryone is false.
+}
+
+// AccessSettingList is a tagged list of AccessSetting
+type AccessSettingList struct {
+	AccessSetting []*AccessSetting `xml:"AccessSetting"`
+}
+
+// LocalSubject is the user or group to which control access settings apply.
+type LocalSubject struct {
+	HREF string `xml:"href,attr"`
+	Name string `xml:"name,attr"`
+	Type string `xml:"type,attr"`
```

Comment:
maybe is worth to mention what types are allowed here. For easy ready in the future